GCRCatalogs/photoz_magerr.py

```diff
@@ -163,7 +241,7 @@ def collect(name, obj):
                     collector.add(name)
             self.handle.visititems(collect)
             self._keys = tuple(collector)
-        return list(self._keys)
+        return list(self._keys) 
```

Comment:
Remove trailing spaces
```suggestion
        return list(self._keys)
```